third_party/cyw30739_sdk/compress_cgs_data.py

```diff
@@ -0,0 +1,256 @@
+#!/usr/bin/env python
+#
+# Copyright 2016-2022, Cypress Semiconductor Corporation (an Infineon company) or
```

Comment:
Code in CHIP should be attributed to CHIP authors, not Cypress/Infineon. I believe it may be dual licensed however it cannot be just infineon.

Reply:
done.